### ISSUE TITLE
`useTypescript: true` issue

### ISSUE THREAD

**Issue by tsanyqudsi** [Subwaytime/vite-aliases#24]
Hi, 

I've noticed the problem when i use `import object from '@path/component'`. Apparently VS Code didn't recognize the module.

After i check, seems like your config generator  forgot to add `/*` for every alias path. Thus we have errors on VS Code.

example : 
the path generated : `"@components" : ["/home/foo/bar/components"]`
what should be generated : `"@components/*" : ["/home/foo/bar/components/*"]`

**Comment by tsanyqudsi** [comment]
oh and, could you prettify the config ? it gets minified with your script.

just a suggestion, maybe you could change

https://github.com/Subwaytime/vite-aliases/blob/89cae48760e4ca8a1aaa37054e0eb501fcc93abb/src/fs/config.ts#L30

to
```
 writeFile(`${file}`, JSON.stringify(data, null, 2), (error) => { 
```

**Comment by Subwaytime** [comment]
Sorry for the late Comment! Github doesnt show activity for me...

I will get it done today, thanks for the report!

**Comment by Subwaytime** [comment]
Fixed in [v0.8.3](https://github.com/Subwaytime/vite-aliases/releases/tag/v0.8.3)
